cloud_fixity/test/index.test.js

```diff
@@ -0,0 +1,61 @@
+// gcloud pubsub topics publish figgy-staging-fixity-request --message '{"preservation_object_id": 1, "file_metadata_node_id": 1, "cloudPath": "example.tif", "md5": "2a28fb702286782b2cbf2ed9a5041ab1"}'
+// gcloud pubsub subscriptions pull --auto-ack figgy-staging-fixity-status
+// gcloud functions deploy checkFixity --runtime nodejs8 --trigger-topic figgy-staging-fixity-request --service-account figgy-staging@pulibrary-figgy-storage-1.iam.gserviceaccount.com
```

Comment:
Can you expand this? it looks like notes on how to initiate the process but could be fleshed out to be more documentation-y

Reply:
I'm actually going to delete them. They were just local notes on how to kick things off before the process actually worked.

Reply:
Okay so then, I'm not sure how everything does get kicked off. I can't find where it pushes to the `figgy-staging-fixity-request` topic. What am I missing?

Reply:
Oh, yeah, that's a new ticket that exists.

Reply:
Created #2919